### PR TITLE
Issue #889 fix for go.mod files

### PR DESCRIPTION
**Pull request by muratmirgun**
#889 Missed go.mod entry fix and import changes

### CODE REVIEW

File: fiber-grpc/go.sum [hunk 1]
```diff
@@ -1,1047 +1,54 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (efectn) [inline fiber-grpc/go.sum:1]: Why did you delete go.sum

Comment (muratmirgun) [reply fiber-grpc/go.sum:1]: go.sum is here maybe something is wrong 
[link](https://github.com/muratmirgun/recipes/tree/master/fiber-grpc)

Comment (muratmirgun) [reply fiber-grpc/go.sum:1]: go.mod version 1.15 -> 1.19 some unused go.sum lines deleted only. go.mod is not fully deleted

Comment (efectn) [reply fiber-grpc/go.sum:1]: Oh sorry 🤦

**Review by efectn — APPROVED** [review]

File: fiber-grpc/go.sum (hunk 1 above)
Comment (muratmirgun) [reply fiber-grpc/go.sum:1]: Thanks Have a nice weekend ✋
